08_project/n0npax/cmd/puppy-server/main_test.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMain(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := "\"Puppy id: 0\""
```

Comment:
Prefer raw strings when character escaping is needed for readability.
Sorry I missed that in the first round feel free to leave it:
```go
expected := `"Puppy id: 0"`
```